### ISSUE TITLE
Why apply the decay to the tokens outside half of the training window instead of the total training window?

### ISSUE THREAD

**Issue by JeffreyYzh** [thu-ml/RIFLEx#31]
Hi~Very nice work of your UltraViCo. 
I am just confused about the depiction in the paper. Sometimes it tells to apply weight decay to the tokens outside the training window, but I find you actually only apply the decay to half of the training window. Why is this? And this implementation can also influence the attention logits within training window, is that right?
Thank you!

**Comment by JeffreyYzh** [comment]
Sorry, I understand the question now — your method treats the current query as the center, masks L/2 tokens on each side, and ultimately applies decay to features outside the training window centered on the query.
